aea/protocols/sample_protocol_specification.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Including a sample is not permitted here, please remove the file.

Reply:
I put it in the example folder for the time being. 
NOTE: make this into a template.